test/fixtures/accounts.js

```diff
@@ -25,7 +25,7 @@ accounts.existingDelegate = {
 	publicKey: 'addb0e15a44b0fdc6ff291be28d8c98f5551d0cd9218d749e30ddb87c6e31ca9',
 	passphrase:
 		'actress route auction pudding shiver crater forum liquid blouse imitate seven front',
-	balance: '0',
+	balance: '1000000000',
```

Comment:
Is this intended?

Reply:
this doesn't seem necessary, removed it!